sql-ballerina/tests/complex-query-test.bal

```diff
@@ -14,6 +14,7 @@
 // under the License.
 
 import ballerina/test;
+//import ballerina/time;
```

Comment:
remove commented line

Reply:
removed in https://github.com/ballerina-platform/module-ballerina-sql/pull/161/commits/1a363a5654c30b1c3b4efd836eb9225713c4e5a6